src/shared/sam/localLambdaRunner.ts

```diff
@@ -348,6 +348,11 @@ export async function runLambdaFunction(
     config: SamLaunchRequestArgs,
     onAfterBuild: () => Promise<void>
 ): Promise<SamLaunchRequestArgs> {
+    // Verify if Docker is running
+    const dockerResponse = await new ChildProcess(true, 'docker', undefined, 'ps').run()
+    if (dockerResponse.exitCode !==0 || dockerResponse.stdout.includes('error during connect')) {
+        throw new Error('Running AWS SAM projects locally requires Docker. Have you got it installed and running?')
```

Comment:
message can be shorter and less colloquial, without any loss of clarity:
```suggestion
        throw new Error('Running AWS SAM projects locally requires Docker. Is it installed and running?')
```